components/Storyblok/SbHomepage.tsx

```diff
@@ -1,20 +1,11 @@
 import { storyblokEditable, type SbBlokData } from '@storyblok/react/rsc';
-import Image from 'next/image';
 import { CreateBloks } from '../CreateBloks';
 import { Heading } from '../Typography';
-import { HomepageHero } from '../Homepage/HomepageHero';
-import { HomepageSplitHero } from '../Homepage/HomepageSplitHero';
-import { Intro } from '../Homepage/Intro';
 import { Masthead } from '../Masthead';
-import { ThemeSection } from '../Homepage/ThemeSection';
-import { FindPurposeSection } from '../Homepage/FindPurposeSection';
-import { ProgressStorySection } from '../Homepage/ProgressStorySection';
-import { Changemaker } from '../Homepage/Changemaker';
-import { IdealFellow } from '../Homepage/IdealFellow';
-import { BlockBanner } from '../Homepage/BlockBanner';
-import { BelowBlockBanner } from '../Homepage/BelowBlockBanner';
-import { getNumBloks } from '@/utilities/getNumBloks';
-import { getProcessedImage } from '@/utilities/getProcessedImage';
+// import { FindPurposeSection } from '../Homepage/FindPurposeSection';
+// import { BlockBanner } from '../Homepage/BlockBanner';
+// import { BelowBlockBanner } from '../Homepage/BelowBlockBanner';
+// import { getProcessedImage } from '@/utilities/getProcessedImage';
```

Comment:
Remove the comments here too if you're not using them.

Reply:
All removed 👍🏼